src/components/Header/Header.tsx

```diff
@@ -1,19 +1,37 @@
-import { AppBar, Toolbar, Button, Box } from '@mui/material';
-import { useNavigate } from 'react-router-dom';
+import { AppBar, Toolbar, Box, Link } from '@mui/material';
 import Logo from '../Logo/Logo';
 import Navigation from '../Navigation/Navigation';
+import { spacings } from '../DesignSystem/DesignSystem';
+import { Link as RouterLink } from 'react-router-dom';
 
-export default function Header() {
-  const navigate = useNavigate();
+interface HeaderProps {
+  siteTree: {
+    label: string;
```

Comment:
Can we specify a type for the site tree so we can import that type here?

Reply:
✅ => 🏷️ ADD sidebar type